superset-frontend/webpack.config.js

```diff
@@ -180,7 +180,7 @@ const config = {
     explore: addPreamble('/src/explore/index.jsx'),
     dashboard: addPreamble('/src/dashboard/index.jsx'),
     sqllab: addPreamble('/src/SqlLab/index.tsx'),
-    welcome: addPreamble('/src/welcome/index.tsx'),
+    crud_views: addPreamble('/src/views/index.tsx'),
```

Comment:
any reason not to follow the camelCase pattern here?

Reply:
Sine these are consumed on the python side, I thought snake case made sense. Realizing now that there's another bundle that's camelCased

Reply:
switched to camelCase for consistency